great_expectations/render/renderer_configuration.py

```diff
@@ -141,20 +152,18 @@ class _RendererParamBase(BaseModel):
             but it is dynamically renamed in order for the RendererParams attribute to have the same name as the param.
         """
 
-        renderer_schema: Dict[str, RendererSchemaType] = Field(
-            ..., allow_mutation=False
-        )
+        renderer_param_schema: RendererSchema = Field(..., allow_mutation=False)
         value: Any = Field(..., allow_mutation=False)
 
         class Config:
             validate_assignment = True
             arbitrary_types_allowed = True
 
         @root_validator(pre=True)
-        def _validate_schema_matches_value(cls, values: dict) -> dict:
-            schema_type: RendererSchemaType = values["schema"]["type"]
+        def _validate_param_type_matches_value(cls, values: dict) -> dict:
+            param_type: RendererValueType = values["schema"]["type"]
             value: Any = values["value"]
-            if schema_type is RendererSchemaType.STRING:
+            if param_type is RendererValueType.STRING:
```

Comment:
```suggestion
            if param_type == RendererValueType.STRING:
```

Reply:
@Kilo59 I went ahead and changed it for added safety, but I think pydantic should already be enforcing this.